src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -263,6 +264,11 @@
 					.examples("on tool break:")
 					.since("2.1.1");
 		}
+		Skript.registerEvent("Item Damage", SimpleEvent.class, PlayerItemDamageEvent.class, "item damag(e|ing)")
+				.description("Called when a player damages an item when using it or armor when taking damage, ie: mining, chopping wood, using shears on a sheep or a zombie attacking you.")
```

Comment:
The description is hard to read and contains a few grammatical mistakes. Using a few sentences instead of just one might help.

Reply:
Ok, Ill change it up here in a bit.

Reply:
Description has been updated.